cli/cmd/sync_test.go

```diff
@@ -204,17 +204,40 @@ func TestSync(t *testing.T) {
 	}
 }
 
+type syncSummaryTable struct {
```

Comment:
Needed a separate struct to deserialize the summary table as the schema is different from the `summary-location` file.

The table schema has internal columns prefixed with `_cq` and the `summary-location` JSON doesn't